src/controllerclientimpl.cpp

```diff
@@ -1914,7 +1925,6 @@ void ControllerClientImpl::_UploadFileToControllerViaForm(std::istream& inputStr
         throw MUJIN_EXCEPTION_FORMAT0("failed to rewind inputStream", MEC_InvalidArguments);
     }
 
-    CURL_OPTION_SAVE_SETTER(_curl, CURLOPT_READFUNCTION, NULL, _ReadIStreamCallback);
```

Comment:
Where did this go?

Reply:
This option is not used when using httppost (as you can see other form uploads does not use this)